### DISCUSSION TITLE
Ressource for plugin creation

### DISCUSSION THREAD

**Discussion by olopost** [pocketbase/pocketbase#2437] [Q&A]
Hi everyone,

I love your project and I wish to build on an extension to publish the creation of a collection as an RSS flow.

Can you direct me on these resources and provide me best practice ?

Best regards,

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand your use case, but if you haven't already, you can check the [Web APIs docs section](https://pocketbase.io/docs/api-records/) for details and examples about the exposed HTTP endpoints (eg. [Collection APIs](https://pocketbase.io/docs/api-collections/)).
